charts/keycloak-operator/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: keycloak-operator
 description: Deploy Keycloak Operator and Keycloak
 type: application
-version: 0.2.4
-appVersion: "22.0.5"
+version: 1.0.0
```

Comment:
1.0.0 :partying_face: 

we might as well also add an icon while we're at it, wdyt?

```yaml
icon: https://www.keycloak.org/resources/images/logo-stacked.svg
```